### ISSUE TITLE
Insert text repeatedly

### ISSUE THREAD

**Issue by dmfilipenko** [atom/vim-mode#859]
Command `3i` don't repeat entered text 3 times.

**Comment by jacekkopecky** [comment]
Duplicate of #580, addressed in #609 which is waiting on the resolution of #568 v #849.

**Comment by 50Wliu** [comment]
Thanks @jacekkopecky.
